CMakeLists.txt

```diff
@@ -28,6 +28,13 @@ foreach(pkg IN LISTS dependencies_pkgs)
 endforeach()
 
 nav2_package()
+add_compile_options(-O3 -mavx2 -mfma -finline-limit=1000000 -ffp-contract=fast)
+
+include_directories(
+  include
+  ${xsimd_INCLUDE_DIRS}
+  ${OpenMP_INCLUDE_DIRS}
+)
 
```

Comment:
target approach more preferred. Why do we need this ?

Reply:
Every file uses xtensor and wants xsimd or openmp optimizations. These seem rather global to me

Reply:
let's stick to target approach